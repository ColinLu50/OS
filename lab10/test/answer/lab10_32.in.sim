43512
1081
1363
820
1331
802
