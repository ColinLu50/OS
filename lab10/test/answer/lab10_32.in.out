FCFS
543 -> 820 -> 491 -> 299 -> 153 -> 217 -> 59 -> 207 -> 127 -> 376 -> 273 -> 16 -> 672 -> 301 -> 560 -> 36 -> 364 -> 227 -> 379 -> 183 -> 487 -> 435 -> 652 -> 26 -> 29 -> 578 -> 635 -> 235 -> 454 -> 319 -> 597 -> 636 -> 201 -> 759 -> 617 -> 298 -> 81 -> 149 -> 366 -> 359 -> 450 -> 110 -> 73 -> 326 -> 562 -> 116 -> 472 -> 763 -> 352 -> 375 -> 545 -> 685 -> 292 -> 110 -> 590 -> 638 -> 467 -> 86 -> 756 -> 605 -> 435 -> 677 -> 99 -> 330 -> 233 -> 420 -> 810 -> 628 -> 646 -> 74 -> 127 -> 657 -> 581 -> 167 -> 135 -> 394 -> 257 -> 101 -> 99 -> 739 -> 555 -> 387 -> 63 -> 88 -> 812 -> 794 -> 229 -> 138 -> 315 -> 681 -> 271 -> 818 -> 344 -> 110 -> 129 -> 753 -> 182 -> 140 -> 443 -> 773 -> 527 -> 476 -> 791 -> 560 -> 257 -> 752 -> 665 -> 340 -> 580 -> 20 -> 114 -> 683 -> 743 -> 369 -> 293 -> 504 -> 591 -> 35 -> 370 -> 110 -> 154 -> 754 -> 123 -> 317 -> 783 -> 426 -> 707 -> 790 -> 374 -> 267 -> 19 -> 398 -> 562 -> 810 -> 348 -> 472 -> 80 -> 198 -> 309 -> 642 -> 95 -> 57 -> 602 -> 509 -> 632 -> 165 -> 476 -> 809 -> 372 -> 76 -> 201 -> 68 -> 513 -> 740 -> 222 -> 467 -> 355 -> 33 -> 430 -> 487 -> 764 -> 510 -> 433
43512
SSTF
543 -> 545 -> 555 -> 560 -> 560 -> 562 -> 562 -> 578 -> 580 -> 581 -> 590 -> 591 -> 597 -> 602 -> 605 -> 617 -> 628 -> 632 -> 635 -> 636 -> 638 -> 642 -> 646 -> 652 -> 657 -> 665 -> 672 -> 677 -> 681 -> 683 -> 685 -> 707 -> 739 -> 740 -> 743 -> 752 -> 753 -> 754 -> 756 -> 759 -> 763 -> 764 -> 773 -> 783 -> 790 -> 791 -> 794 -> 809 -> 810 -> 810 -> 812 -> 818 -> 820 -> 527 -> 513 -> 510 -> 509 -> 504 -> 491 -> 487 -> 487 -> 476 -> 476 -> 472 -> 472 -> 467 -> 467 -> 454 -> 450 -> 443 -> 435 -> 435 -> 433 -> 430 -> 426 -> 420 -> 398 -> 394 -> 387 -> 379 -> 376 -> 375 -> 374 -> 372 -> 370 -> 369 -> 366 -> 364 -> 359 -> 355 -> 352 -> 348 -> 344 -> 340 -> 330 -> 326 -> 319 -> 317 -> 315 -> 309 -> 301 -> 299 -> 298 -> 293 -> 292 -> 273 -> 271 -> 267 -> 257 -> 257 -> 235 -> 233 -> 229 -> 227 -> 222 -> 217 -> 207 -> 201 -> 201 -> 198 -> 183 -> 182 -> 167 -> 165 -> 154 -> 153 -> 149 -> 140 -> 138 -> 135 -> 129 -> 127 -> 127 -> 123 -> 116 -> 114 -> 110 -> 110 -> 110 -> 110 -> 101 -> 99 -> 99 -> 95 -> 88 -> 86 -> 81 -> 80 -> 76 -> 74 -> 73 -> 68 -> 63 -> 59 -> 57 -> 36 -> 35 -> 33 -> 29 -> 26 -> 20 -> 19 -> 16
1081
SCAN
543 -> 527 -> 513 -> 510 -> 509 -> 504 -> 491 -> 487 -> 476 -> 472 -> 467 -> 454 -> 450 -> 443 -> 435 -> 433 -> 430 -> 426 -> 420 -> 398 -> 394 -> 387 -> 379 -> 376 -> 375 -> 374 -> 372 -> 370 -> 369 -> 366 -> 364 -> 359 -> 355 -> 352 -> 348 -> 344 -> 340 -> 330 -> 326 -> 319 -> 317 -> 315 -> 309 -> 301 -> 299 -> 298 -> 293 -> 292 -> 273 -> 271 -> 267 -> 257 -> 235 -> 233 -> 229 -> 227 -> 222 -> 217 -> 207 -> 201 -> 198 -> 183 -> 182 -> 167 -> 165 -> 154 -> 153 -> 149 -> 140 -> 138 -> 135 -> 129 -> 127 -> 123 -> 116 -> 114 -> 110 -> 101 -> 99 -> 95 -> 88 -> 86 -> 81 -> 80 -> 76 -> 74 -> 73 -> 68 -> 63 -> 59 -> 57 -> 36 -> 35 -> 33 -> 29 -> 26 -> 20 -> 19 -> 16 -> 0 -> 545 -> 555 -> 560 -> 562 -> 578 -> 580 -> 581 -> 590 -> 591 -> 597 -> 602 -> 605 -> 617 -> 628 -> 632 -> 635 -> 636 -> 638 -> 642 -> 646 -> 652 -> 657 -> 665 -> 672 -> 677 -> 681 -> 683 -> 685 -> 707 -> 739 -> 740 -> 743 -> 752 -> 753 -> 754 -> 756 -> 759 -> 763 -> 764 -> 773 -> 783 -> 790 -> 791 -> 794 -> 809 -> 810 -> 812 -> 818 -> 820
1363
C-SCAN
543 -> 527 -> 513 -> 510 -> 509 -> 504 -> 491 -> 487 -> 476 -> 472 -> 467 -> 454 -> 450 -> 443 -> 435 -> 433 -> 430 -> 426 -> 420 -> 398 -> 394 -> 387 -> 379 -> 376 -> 375 -> 374 -> 372 -> 370 -> 369 -> 366 -> 364 -> 359 -> 355 -> 352 -> 348 -> 344 -> 340 -> 330 -> 326 -> 319 -> 317 -> 315 -> 309 -> 301 -> 299 -> 298 -> 293 -> 292 -> 273 -> 271 -> 267 -> 257 -> 235 -> 233 -> 229 -> 227 -> 222 -> 217 -> 207 -> 201 -> 198 -> 183 -> 182 -> 167 -> 165 -> 154 -> 153 -> 149 -> 140 -> 138 -> 135 -> 129 -> 127 -> 123 -> 116 -> 114 -> 110 -> 101 -> 99 -> 95 -> 88 -> 86 -> 81 -> 80 -> 76 -> 74 -> 73 -> 68 -> 63 -> 59 -> 57 -> 36 -> 35 -> 33 -> 29 -> 26 -> 20 -> 19 -> 16 -> 0 -> 822 -> 820 -> 818 -> 812 -> 810 -> 809 -> 794 -> 791 -> 790 -> 783 -> 773 -> 764 -> 763 -> 759 -> 756 -> 754 -> 753 -> 752 -> 743 -> 740 -> 739 -> 707 -> 685 -> 683 -> 681 -> 677 -> 672 -> 665 -> 657 -> 652 -> 646 -> 642 -> 638 -> 636 -> 635 -> 632 -> 628 -> 617 -> 605 -> 602 -> 597 -> 591 -> 590 -> 581 -> 580 -> 578 -> 562 -> 560 -> 555 -> 545
820
LOOK
543 -> 527 -> 513 -> 510 -> 509 -> 504 -> 491 -> 487 -> 476 -> 472 -> 467 -> 454 -> 450 -> 443 -> 435 -> 433 -> 430 -> 426 -> 420 -> 398 -> 394 -> 387 -> 379 -> 376 -> 375 -> 374 -> 372 -> 370 -> 369 -> 366 -> 364 -> 359 -> 355 -> 352 -> 348 -> 344 -> 340 -> 330 -> 326 -> 319 -> 317 -> 315 -> 309 -> 301 -> 299 -> 298 -> 293 -> 292 -> 273 -> 271 -> 267 -> 257 -> 235 -> 233 -> 229 -> 227 -> 222 -> 217 -> 207 -> 201 -> 198 -> 183 -> 182 -> 167 -> 165 -> 154 -> 153 -> 149 -> 140 -> 138 -> 135 -> 129 -> 127 -> 123 -> 116 -> 114 -> 110 -> 101 -> 99 -> 95 -> 88 -> 86 -> 81 -> 80 -> 76 -> 74 -> 73 -> 68 -> 63 -> 59 -> 57 -> 36 -> 35 -> 33 -> 29 -> 26 -> 20 -> 19 -> 16 -> 545 -> 555 -> 560 -> 562 -> 578 -> 580 -> 581 -> 590 -> 591 -> 597 -> 602 -> 605 -> 617 -> 628 -> 632 -> 635 -> 636 -> 638 -> 642 -> 646 -> 652 -> 657 -> 665 -> 672 -> 677 -> 681 -> 683 -> 685 -> 707 -> 739 -> 740 -> 743 -> 752 -> 753 -> 754 -> 756 -> 759 -> 763 -> 764 -> 773 -> 783 -> 790 -> 791 -> 794 -> 809 -> 810 -> 812 -> 818 -> 820
1331
C-LOOK
543 -> 527 -> 513 -> 510 -> 509 -> 504 -> 491 -> 487 -> 476 -> 472 -> 467 -> 454 -> 450 -> 443 -> 435 -> 433 -> 430 -> 426 -> 420 -> 398 -> 394 -> 387 -> 379 -> 376 -> 375 -> 374 -> 372 -> 370 -> 369 -> 366 -> 364 -> 359 -> 355 -> 352 -> 348 -> 344 -> 340 -> 330 -> 326 -> 319 -> 317 -> 315 -> 309 -> 301 -> 299 -> 298 -> 293 -> 292 -> 273 -> 271 -> 267 -> 257 -> 235 -> 233 -> 229 -> 227 -> 222 -> 217 -> 207 -> 201 -> 198 -> 183 -> 182 -> 167 -> 165 -> 154 -> 153 -> 149 -> 140 -> 138 -> 135 -> 129 -> 127 -> 123 -> 116 -> 114 -> 110 -> 101 -> 99 -> 95 -> 88 -> 86 -> 81 -> 80 -> 76 -> 74 -> 73 -> 68 -> 63 -> 59 -> 57 -> 36 -> 35 -> 33 -> 29 -> 26 -> 20 -> 19 -> 16 -> 820 -> 818 -> 812 -> 810 -> 809 -> 794 -> 791 -> 790 -> 783 -> 773 -> 764 -> 763 -> 759 -> 756 -> 754 -> 753 -> 752 -> 743 -> 740 -> 739 -> 707 -> 685 -> 683 -> 681 -> 677 -> 672 -> 665 -> 657 -> 652 -> 646 -> 642 -> 638 -> 636 -> 635 -> 632 -> 628 -> 617 -> 605 -> 602 -> 597 -> 591 -> 590 -> 581 -> 580 -> 578 -> 562 -> 560 -> 555 -> 545
802
